src/go/pt-k8s-pxc-recovery/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"flag"
+	"log"
+
+	"github.com/percona/percona-toolkit/src/go/pt-k8s-pxc-recovery/helpers"
+	"github.com/percona/percona-toolkit/src/go/pt-k8s-pxc-recovery/recover"
+)
+
+func stepOrError(err error) {
+	if err != nil {
+		log.Fatal(err)
```

Comment:
make obvious for the user that this is an error, e. g. add "Error: " prefix to the output

Reply:
fixed